src/bento/cartData.js

```diff
@@ -30,17 +16,14 @@ export const myFilesPageData = {
   wizardIconSrc: 'https://raw.githubusercontent.com/CBIIT/bento-frontend/master/src/assets/icons/Cart-Wizard-Step3.svg',
   wizardIconAlt: 'Bento MyFiles Wizard',
   manifestFileName: 'BENTO File Manifest',
+  actionMessage: {
```

Comment:
Looks like some change in the custodian file  is it necessary as we need to keep as similar as possible with BENTO?

Reply:
This is only a recommendation. You can ignore it if there is a better way.

Reply:
I agree this is a better way than the previous one. Do we want to update the docs to represent this?

Reply:
As we are the custodian for ICDC. I think. we can go ahead and merge this